Samurai/Tasks/TippingManager.cs

```diff
@@ -2,22 +2,16 @@
 using System.Collections.Generic;
 using System.Diagnostics;
 using System.Linq;
-using System.Net;
-using System.Net.Http;
-using System.Net.Http.Headers;
-using System.Text;
 using System.Threading;
 using System.Threading.Tasks;
+using AcidChicken.Samurai.Models;
```

Comment:
All of the local namespace reference instructions have to be included in the local namespace block.